### DISCUSSION TITLE
v1.1.14

### DISCUSSION THREAD

**Discussion by n4ze3m** [n4ze3m/page-assist#133] [Announcements]
## What's Changed
* Ability to add custom headers to Ollama URL via advanced URL mode #106 
* [NEW] Chrome AI support 

**Full Changelog**: https://github.com/n4ze3m/page-assist/compare/v1.1.13...v1.1.14

<hr /><em>This discussion was created from the release <a href='https://github.com/n4ze3m/page-assist/releases/tag/v1.1.14'>v1.1.14</a>.</em>

**Comment by lambirou** [comment]
I am very glad that you took my suggestion into consideration.

**Comment by Mr-hackerman** [comment]
Best front end out there. Works really well.
